### ISSUE TITLE
remove warnings 

### ISSUE THREAD

**Issue by cht8687** [cht8687/react-expandable-listview#10]
EventPluginUtils.js:151 Warning: a `div` tag (owner: `Motion`) was passed a numeric string value for CSS property `height` (value: `0`) which will be treated as a unitless number in a future version of React.

**Comment by thedanielvogelsang** [comment]
Yes I'm receiving the same issue, due to the numeric height value prop dependency; I've tried changing it to: height: "600px" but then it breaks. Please fix!

**Comment by cht8687** [comment]
@thedanielvogelsang 
Hi, do you have any solutions to this issue? if you can send any PR that would be highly appreciated.
